examples/webgl_physics_rope.html

```diff
@@ -35,7 +35,7 @@
 	<script src="../build/three.js"></script>
 	<script src="js/libs/ammo.js"></script>
 	<script src="js/controls/OrbitControls.js"></script>
-        <script src="js/Detector.js"></script>
+  <script src="js/Detector.js"></script>
```

Comment:
Minor O:)

Reply:
Ups, there were still some soft tabs in that file.